### PR TITLE
Steph - event/user/review schemas

### PR DESCRIPTION
**Pull request by Stephc0211**

### CODE REVIEW

**Comment by YANLING-H** [comment]
remember to add labels and projects

**Comment by YANLING-H** [comment]
remember to merge your feature branch to the develop branch instead of master branch

File: backend/src/models/event.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        capacity: { type: Number, requried: true },
+        fee_detail: { type: String, default: "" },
+        description: { type: String, length: 200, default: "" },
+        price: { type: Number, default: 0 },
+        hostid: { type: [{ address: String, id: Number }], required: true},
+/*         host_user_obj_id
+        participant_obj_id */
+    },
+
+    /* {
```
Comment (MuJiongan) [inline backend/src/models/event.js:19]: could you remove all the timestamps?

File: backend/src/models/reviews.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+const reviewSchema = new Schema(
+    {
+        userid: { type: String, required:true },
```
Comment (MuJiongan) [inline backend/src/models/reviews.js:6]: inconsistency between the space between colon and word. Should be `required: true` with a space in between

File: backend/src/models/user.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema;
+
+
+const userSchema = new Schema(
+    {
+        nickname: { type: String, required: true },
+        name: { type: String, required: true },
+        pronoun: { type: String, deafault: "" },
```
Comment (MuJiongan) [inline backend/src/models/user.js:9]: there is a typo `deafault`

File: backend/src/models/event.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        name: { type: String, required: true },
+        location: { type: String, required: true },
+        start_time: { type: Date, required: true },
+        end_time: { type: Date, required: true },
+        capacity: { type: Number, requried: true },
+        fee_detail: { type: String, default: "" },
+        description: { type: String, length: 200, default: "" },
+        price: { type: Number, default: 0 },
+        hostid: { type: [{ address: String, id: Number }], required: true},
+/*         host_user_obj_id
```
Comment (MuJiongan) [inline backend/src/models/event.js:15]: could u remove all the comments?

**Pull request merged**